nifi-commons/nifi-sensitive-property-provider/src/test/java/org/apache/nifi/properties/AWSSensitivePropertyProviderIT.java

```diff
@@ -121,9 +121,9 @@ public void testEncryptDecrypt() {
 
     private static void runEncryptDecryptTest() {
         logger.info("Plaintext: " + SAMPLE_PLAINTEXT);
-        String protectedValue = spp.protect(SAMPLE_PLAINTEXT);
+        String protectedValue = spp.protect(SAMPLE_PLAINTEXT, null);
         logger.info("Protected Value: " + protectedValue);
-        String unprotectedValue = spp.unprotect(protectedValue);
+        String unprotectedValue = spp.unprotect(protectedValue, null);
```

Comment:
Although this particular implementation does not use the PropertyContext, it looks like other tests use an actual value as opposed to `null`. It seems that it would be better to use a static PropertyContext in the test class to avoid implying that the PropertyContext can be null.